HW28/app.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask, render_template
+import random
+
+app = Flask(__name__)
+
+# list of cat images
+images = [
+    "https://media.tenor.com/t0eFbcxLGgIAAAAC/fat-cat-laser-eyes.gif",
+    "https://media.tenor.com/pOS38tUuVnQAAAAd/cat-meme.gif",
+    "https://media.tenor.com/lQlIBQeeruwAAAAd/wanted-cat.gif",
+]
+
+
+@app.route("/")
+def index():
+    url = random.choice(images)
+    return render_template("index.html", url=url)
```

Comment:
It will not work for someone else, because there is no `index.html` in the repository